docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -78,13 +82,26 @@ $ tctl create -f access.yaml
 
 Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
 
-```bash
-$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=720h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuring-teleport-slack).
-
+<Tabs>
+<TabItem label="Teleport Cloud or via Proxy connection">
+  If you are using Teleport Cloud or want to have the plugin only 
```

Comment:
...or want to have the plugin only what?

Reply:
corrected